core/pkg/opencost/allocation_json.go

```diff
@@ -84,8 +85,8 @@ func (aj *AllocationJSON) BuildFromAllocation(a *Allocation) {
 	aj.CPUCostIdle = formatFloat64ForResponse(a.CPUCostIdle)
 	aj.CPUEfficiency = formatFloat64ForResponse(a.CPUEfficiency())
 	aj.GPUCount = formatFloat64ForResponse(a.GPUs())
-	aj.GPURequestAverage = formatFloat64ForResponse(a.GPURequestAverage)
-	aj.GPUUsageAverage = formatFloat64ForResponse(a.GPUUsageAverage)
+	aj.GPURequestAverage = formatFloat64ForResponse(a.deprecatedGPURequestAverage)
+	aj.GPUUsageAverage = formatFloat64ForResponse(a.deprecatedGPUUsageAverage)
```

Comment:
Not sure about these fields? Should we get the values here from the new data, or rather, we should remove/omit these fields from this `AllocationJSON` object and add a new structure representing the new data field?

Reply:
We already added a new structure representing new data in Ln #125.

This fields, I can just delete since we already have "-" in the json for ln 29-30.. 

Thoughts @mbolt35